docs/en_US/Blog/HPOComparison.md

```diff
@@ -0,0 +1,323 @@
+# Hyperparameter Optimization Comparison
+*Posted by Anonymous Author*
+
+Comparison of Hyperparameter Optimization algorithms on several problems.
+
+Hyperparameter Optimization algorithms are list below:
+
+- Random Search
+- Grid Search
+- Evolution
+- Anneal
+- Metis
+- TPE
+- SMAC
+- HyperBand
+- BOHB
+
+All algorithms run in NNI local environment.
+
+## First Fake Function
+
+### Problem Description
+
+Firstly, one fake function is made to test the effectiveness in convex problem when hyperparameters are not independent.
+
+Fake Function:
+
+![](<http://latex.codecogs.com/gif.latex?Loss(x,y,z)=(x-30+0.1)^2\times(y-40+0.1)^2\times(x+z-50+0.1)^2>)
+
+### Search Space
+
+```json
+{
+  "x": {
+    "_type": "randint",
```

Comment:
using "randint" means you don't care about the order of value.  I suggested you remove this part.

Reply:
I still keep it.